<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Professional</class>
 <widget class="QMainWindow" name="Professional">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Professional</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="r_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color:rgb(232, 0, 0);</string>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="b_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
color:rgb(0, 0, 241);
</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="g_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 223, 0);</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLabel" name="r_value">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 200);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="b_value">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>159</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 200);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="g_value">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 200);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rgb_value">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>341</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rgb_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>341</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>RGB</string>
    </property>
   </widget>
   <widget class="QLabel" name="c_label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>341</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
    </property>
    <property name="text">
     <string>溶液浓度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>341</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upload">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;仿宋&quot;;
border-color: rgb(254, 255, 190);</string>
    </property>
    <property name="text">
     <string>上传图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="cal_show">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rgb_unknown">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>RGB(Unknown)</string>
    </property>
   </widget>
   <widget class="QLabel" name="cal_rgb">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>y=aX+b</string>
    </property>
   </widget>
   <widget class="QLabel" name="show">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="p2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>500</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mol_unknown">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>700</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>mol(Unknown)</string>
    </property>
   </widget>
   <widget class="QLabel" name="cal_mol">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>700</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RGB_Parameter">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>460</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>RGB Parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="mol_Parameter">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>460</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>mol Parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="r2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>770</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>R^2</string>
    </property>
   </widget>
   <widget class="QLabel" name="r2_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>770</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_value">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pic1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mol1">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>510</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="mol2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>540</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="mol3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>570</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="mol4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>600</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="mol5">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>630</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="mol6">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>660</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="p1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(200, 200, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="v5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>630</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="v3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>570</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="v4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>600</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="v6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>660</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="v2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="v1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>510</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="prompt">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>380</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;
color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>请输入浓度！</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
